android/src/main/java/com/guichaguri/trackplayer/service/MusicManager.java

```diff
@@ -197,6 +197,10 @@ public void onTrackUpdate(Track previous, long prevPos, Track next) {
         service.emit(MusicEvents.PLAYBACK_TRACK_CHANGED, bundle);
     }
 
+    public void updateMetadata(Track track) {
+        metadata.updateMetadata(track);
+    }
+
```

Comment:
Please, don't merge another PR that has not been approved yet into yours. There are a few problems with its code that should be fixed first. And also, you're removing @HybridFox's authored commits.

Reply:
The comits created by @HybridFox, I can not get access to merge.

Reply:
You can fork his repository and then merge into yours, that would keep his name in the commits, but I don't think you should do that, as his PR still has problems that must be fixed first.

Reply:
Perfect! I will make the corrections you commented on, in fact some are already done. If you feel better, I will "Cancel" this "Pull Request" and I will send you with the corrections and without the use of static variable.